data/functions.sql

```diff
@@ -556,10 +556,33 @@ RETURNS BOOLEAN AS $$
 BEGIN
     RETURN (
       tags->'type' = 'route' AND
-      tags->'route' IN ('hiking', 'foot') AND
-      tags->'network' IN ('iwn','nwn','rwn','lwn')
+      tags->'route' IN ('hiking', 'foot', 'bicycle') AND
```

Comment:
I included cycling networks here (see related https://github.com/mapzen/vector-datasource/issues/647) because their zoom ranges needed adjusting for the paths to have the right balance. 

I don't think any additional zoom work is need for those with respect to #647:

* `CREATE OR REPLACE FUNCTION mz_is_path_major_route_relation(tags hstore)`
* `CREATE OR REPLACE FUNCTION mz_calculate_path_major_route(osm_id BIGINT)`